src/confd/src/infix-containers.c

```diff
@@ -54,9 +53,12 @@ static int add(const char *name, struct lyd_node *cif)
 	 * command ensures any already running container is stopped and
 	 * deleted so that it releases all claimed resources.
 	 */
+	image = lydx_get_cattr(cif, "image");
 	fprintf(fp, "#!/bin/sh\n"
+		"# meta-name: %s\n"
+		"# meta-image: %s\n"
```

Comment:
```suggestion
		"export CONTAINER_NAME=\"%s\"\n"
		"export CONTAINER_IMAGE=\"%s\"\n"
```
How about passing this info in the environment?

Reply:
Good idea, but does not work in this case since it's not this script that needs the info.  Instead it's for the `container upgrade foo` command, which goes dumpster diving for the container image.